metrics-model-libs/safety/definition/safety.md

```diff
@@ -1,191 +1,42 @@
-|Source:  ‘Understand’ section of Metric Inclusive Leadership - CHAOSS|
-|Share how this went for you:|
-```
-Examples of leadership include:
-Individuals serving as project maintainers (sometimes referred to as owners)
-Individuals with repository merge access
-Individuals who are organization members for the repository
-Individuals who have defined community roles (community representatives, community speakers)
-Individuals listed in project documents as contacts for issues with builds, documentation, or other community concerns
-Individuals serving as community event organizers
-Individuals serving as community mentors for available mentorship programs
-```
-
-
-
-|Inclusive Governance - Leadership Data Analysis|
-| ------ |
-|Difficulty: Challenging|
-|Audience: Community Manager/Maintainer, OSPO|
-|Description: Track non-team member contributions to leadership tasks and associated sentiment , as a measure of how well a project is empowering(or not empowering) new leadership. As those in leadership roles, also hold influence and set the tone for what is, and what is not acceptable it is also important to track sentiment/behaviors. | 
-|Source: GitHub Discussions Community Insights responses to new contributors.|
-
-
-## [Board/Council Diversity](https://chaoss.community/metric-board-council-diversity/)
-|Leadership/Board/Council Diversity  - Observation Checklist|
-| ------ |
-|Difficulty: Easy|
-|Audience: Contributor|
-|Description: Observing board/council diversity helps you understand how likely it is that diverse perspectives are understood and prioritized, which has direct influence on safety of protected groups.  | 
-|Source:  (taken from Metric Board Council Diversity - CHAOSS)|
-|Observe whether open elections are held to see if candidates are representative of the member base. (Note: elections typically favor candidates of majorities and do not ensure that minorities are represented after the election)     Observe the diversity of board or council members from the project webpage (limitation: demographics are not always self-evident).   Ask the board for a report of the diversity of the board or council.|
-|Share how this went for you:|
+Note that should evaluations determine that a project is determined not to be set up for success, this is not necessarily a reflection on that project's willingness, or capacity  - rather it’s a tool for learning and improving.  For example, if a funding company determines a project is not set up for equitable governance, they may opt to provide supplementary support as part of a grant to help that project improve.
 
+# Metrics in the Metrics Model 
+### [Psychological Safety](https://chaoss.community/metric-psychological-safety/) 
```

Comment:
Do we have fixed link for these metrics?

Reply:
I just added them 👍 - Thanks @eyehwan!